### PR TITLE
Fix #874: Construct ConvolverNode.

### LINKED ISSUE #874: Property bag for Convolver
An addition to #696.

Constructor:

```
Constructor(BaseAudioContext context, optional ConvolverOptions options)
```

Dictionary:
```
dictionary ConvolverOptions : AudioNodeChannelOptions {
  AudioBuffer buffer;
  boolean normalize = true;
}
```

This is a bit weird because `normalize` defaults to true.  Perhaps we should rename it to `disableNormalization`, defaulting to false.  This matches what we did for `PeriodicWaveConstraints`.

### PR DESCRIPTION
**Pull request by rtoy**
Add constructor and define the options dictionary.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5975,6 +5977,35 @@ <h2 id="ConvolverNode">
...
+            <dt>
+              AudioBuffer buffer
+            </dt>
+            <dd>
+              The desired buffer for the <a><code>ConvolverNode</code></a>.
+              This buffer will be normalized according to the value of
+              <code>disableNormalization</code>.
+            </dd>
+            <dt>
+              boolean disableNormalization = false
```
Comment (joeberkovitz) [inline index.html]: This is opposite in sense to an existing `normalize` attribute on a `ConvolverNode`.  Is this intentional?

Comment (rtoy) [inline index.html]: Yes. I was following the suggestions and decisions made in https://github.com/WebAudio/web-audio-api/pull/545#issuecomment-112599527 wrt to the boolean trap and why we used `disableNormalization` for the `PeriodicWave` dictionary.

**Comment by joeberkovitz** [comment]
cool.

**Comment by rtoy** [comment]
Assuming "cool" means lgtm, I'll fix up the AudioNodeChannelOptions name and merge this soon.

Unless you object.

**Pull request merged**